bundle/edu.gemini.spModel.io/src/main/scala/edu/gemini/spModel/io/impl/migration/to2016A/To2016A.scala

```diff
@@ -59,6 +60,20 @@ object To2016A extends Migration {
       plutoParams.foreach(ps.addParam)
     }
 
+
+  //All unblocked modes in GPI must set useCal = false
+  private def updateGpiUnblockedModes(d: Document): Unit = {
+
+    val isUnblocked = Set("UNBLOCKED_Y", "UNBLOCKED_H", "UNBLOCKED_J", "UNBLOCKED_K1", "UNBLOCKED_K2")
```

Comment:
Yes, I thought about that, but the code ends up being more verbose. As there are not too many of them, I thought it was better to just copy the names I needed, carefully.  The test in `GpiUnblockedModesTest` checks that the names are correct,as it uses the real types in `Gpi`. If there were a typo in any of these, that test would fail, so I'm relatively confident with it.